robocop_ng/cogs/meme.py

```diff
@@ -56,6 +56,19 @@ async def warm_member(self, ctx, user: Optional[discord.Member]):
                 f"({fahrenheit}°F, {kelvin}K)."
             )
 
+    @commands.check(check_if_staff_or_ot)
+    @commands.command(hidden=True, name="lick")
+    async def licky(self, ctx, user: Optional[discord.Member]):
```

Comment:
```suggestion
    @commands.command(hidden=True)
    async def lick(self, ctx, user: Optional[discord.Member]):
```
Just to simplify the syntax a little bit.